sass/vendor/font-awesome/font-awesome.scss

```diff
@@ -0,0 +1,18 @@
+/*!
```

Comment:
This would be a good candidate to move up a level - ie it is the stub. The stub is included by the wildcard *then* it includes its other parts.